manufacturing-client/src/main.rs

```diff
@@ -291,15 +293,57 @@ async fn main() -> Result<()> {
 }
 
 async fn get_mfg_info(mfg_string_type: MfgStringType) -> Result<CborSimpleType> {
-    if let Some(mfg_info) = env::var_os("MANUFACTURING_INFO") {
-        return Ok(CborSimpleType::Text(mfg_info.into_string().unwrap()));
-    }
     match mfg_string_type {
         MfgStringType::SerialNumber => {
```

Comment:
I think that we could define a `MfgStringType::MAC` type so that people can directly specify if they want the MAC address as the identifier in the `manufacturing_server.yml` config file. The MAC would still be the fallback when no serial number is found, but giving the users this explicit option would make things more clear.

Reply:
I think we shouldn't fallback, if the user asks for SerialNumber they get the old behavior - if they ask for the new one they get the new info which may or may not include the SN. I think we should call it something like `MfgStringType::Extended` or something like that, the point is this new format allows us to add fields to the mfgstring in a backwards compatible way too